server/jetstream_benchmark_test.go

```diff
@@ -1174,3 +1174,133 @@ func BenchmarkJetStreamKV(b *testing.B) {
 		)
 	}
 }
+
+func BenchmarkJetStreamObjStore(b *testing.B) {
+	const (
+		verbose            = false
+		objStoreNamePrefix = "B_"
+		keyPrefix          = "K_"
+		seed               = 12345
+		init               = true
+	)
+
+	benchmarksCases := []struct {
+		clusterSize int
+		replicas    int
+		numKeys     int
+		minBytes    int
+		maxBytes    int
+		rwRatio     float64
+	}{
```

Comment:
I want to add a couple more dimensions so this table gets large and unwieldy.

How about splitting the following dimensions:
```
Cluster size == replicas : 1, 3
keys: 10, 1000, 100'000
min, max object size: <10, 10>, <1k, 1k>, <100k, 100k>, <10M, 10M>, <10, 10M>
RWRatio: 0.0, 1.0, 0.8 (80% read)
Storage: mem, file
```

Then having sub tests with all possible permutations (for nightly benchmarks, we'd filter what runs, e.g., N=3 with file storage), but other folks using the benchmarks may want to run other cases in some cases.

Example test names:
`BenchmarkJetStreamObjStore/N=3,R=3/FileStore/K=1000,minSize=1000,maxSize=1000/readonly`
`BenchmarkJetStreamObjStore/N=3,R=3/MemStore/K=1000,minSize=1000,maxSize=1000/rw=0.8`